utils/build-script

```diff
@@ -18,6 +18,8 @@ import platform
 import sys
 import time
 
+import six
```

Comment:
Looks like this needs to go between the `build_swift` and `swift_build_support` imports. You can run the Python lint locally by running `python utils/python_lint.py`.